lib/galaxy/tool_util/deps/mulled/util.py

```diff
@@ -13,6 +14,9 @@
 
 log = logging.getLogger(__name__)
 
+BUILD_NUMBER_REGEX = re.compile(r'\d+$')
+MULLED_TAG_CACHE = collections.defaultdict(dict)
```

Comment:
Another difference, I removed this line since `MULLED_TAG_CACHE` is now unused.

Reply:
Ah yeah, I deleted that one now.